gym_cap/envs/agent.py

```diff
@@ -25,12 +25,12 @@ def __init__(self, loc, map_only, team_number):
         """
         self.isAlive = True
         self.x, self.y = loc
-        self.length, self.breath = map_size
+        self.length, self.breath = map_only.shape
         self.step = UGV_STEP
         self.range = UGV_RANGE
         self.a_range = UGV_A_RANGE
         self.air = False
-        self.memory = np.array([[-1 for i in range(self.length)] for j in range(self.breath)])
+        self.memory = np.full((self.length, self.breath), -1)
         self.mode = "fog"
```

Comment:
Set default to None

Reply:
set default value of memory_mode to None